src/components/multiselect.vue

```diff
@@ -49,13 +49,16 @@ except according to the terms contained in the LICENSE file.
           :class="{ 'shows-all': searchValue === '' }" @change="changeCheckbox">
           <template v-if="options != null">
             <!-- eslint-disable-next-line vue/object-curly-newline -->
-            <li v-for="({ value, key = value, text = value, title = text }, i) in options"
+            <li v-for="({ value, key = value, text = value, description }, i) in options"
               :key="key" :class="{ 'search-match': searchMatches.has(value) }">
               <div class="checkbox">
                 <label>
-                  <input type="checkbox" :data-index="i">
-                  <span :title="title">{{ text }}</span>
+                  <input type="checkbox" :data-index="i"
+                    :aria-describedby="description != null ? descriptionId(i) : null">
+                  <span v-if="description == null" v-tooltip.text>{{ text }}</span>
+                  <span v-else v-tooltip.no-aria="description">{{ text }}</span>
```

Comment:
In `SubmissionDecrypt`, the tooltip is over the checkbox as a whole: both the label text and the checkbox `<input>`. Here, we only show the tooltip over the text. That's how we had it set up before and I think is a reasonable user experience.